grype_check2.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+IMAGES=$(kubectl get pods --all-namespaces -o json | jq -r '.items[].spec.containers[].image' | sort |uniq)
+
+SYFT_BIN="podman run -v ${PWD}:/workdir -w /workdir docker.io/anchore/syft:latest"
```

Comment:
Very clever to leave `SYFT_BIN` in.

Reply:
It's not really that clever given this should be checked and is clearly defined in Anchore's policy.